src/main/java/org/folio/circulation/domain/ConfigurationRepository.java

```diff
@@ -62,6 +67,15 @@ private LoanAnonymizationConfiguration getFirstConfiguration(Collection<Configur
     return LoanAnonymizationConfiguration.from(new JsonObject(period));
   }
 
+  private Result<Response> refuseWhenNoConfigurationsFound(Response response) {
+    if (response.getJson().getInteger("totalRecords") == 0) {
```

Comment:
There doesn't appear to be a test demonstrating that this check resolves the issue identified, please can one be added.

Reply:
The test have been added.